ietf/doc/views_review.py

```diff
@@ -438,7 +440,7 @@ def withdraw_reviewer_assignment(request, name, assignment_id):
         )            
 
         policy = get_reviewer_queue_policy(review_assignment.review_request.team)
-        policy.return_reviewer_to_rotation_top(review_assignment.reviewer.person)
+        policy.return_reviewer_to_rotation_top(review_assignment.reviewer.person, True)
```

Comment:
With the change I suggested in the top level comments, you could leave this line unchanged (don't pass the second argument). If you want to pass it explicitly, use the named instead of positional form please.

Reply:
So I had already added the wants_to_be_next to the test_policies.py, but the tests still fail. Thats why I asked yesterday, whether you wanted me to commit those changes to the github, even when I know the test will fail. 

Before adding them there, they failed with a line number, so it was easy to add them there, now when I added those parametrs they simply fail saying type error and no line number:

`dev ➜ /workspace $ ietf/manage.py test --skip-coverage --no-validate-html --traceback --settings=settings_postgrestest  ietf.review.tests_policies.LeastRecentlyUsedReviewerQueuePolicyTest.test_return_reviewer_to_rotation_top ietf.review.tests_policies.RotateAlphabeticallyReviewerQueuePolicyTest.test_return_reviewer_to_rotation_top
     Datatracker 8.0.0-dev test suite, 27 March 2023 02:01:29 PDT:
     Python 3.9.16 (main, Feb  4 2023, 12:10:12)  [GCC 10.2.1 20210110].
     Django 2.2.28, settings 'settings_postgrestest'
     Changing TEMPLATES[0]['OPTIONS']['string_if_invalid'] to '' during testing
     Changing INTERNAL_IPS to '[]' during testing.
     Running an SMTP test server on 127.0.0.1:2026 to catch outgoing email.
     Loading factory-boy random state from .factoryboy_random_state
     Not validating any generated HTML; please do so at least once before committing changes
     Creating test database...
     Loading global test fixtures: names, ietf.utils.test_data.make_immutable_base_data, nomcom_templates, proceedings_templates
System check identified no issues (1 silenced).
EE
======================================================================
ERROR: test_return_reviewer_to_rotation_top (ietf.review.tests_policies.LeastRecentlyUsedReviewerQueuePolicyTest.test_return_reviewer_to_rotation_top)
----------------------------------------------------------------------
TypeError: test_return_reviewer_to_rotation_top() missing 2 required positional arguments: 'reviewer' and 'wants_to_be_next'

======================================================================
ERROR: test_return_reviewer_to_rotation_top (ietf.review.tests_policies.RotateAlphabeticallyReviewerQueuePolicyTest.test_return_reviewer_to_rotation_top)
----------------------------------------------------------------------
TypeError: test_return_reviewer_to_rotation_top() missing 2 required positional arguments: 'person' and 'wants_to_be_next'

----------------------------------------------------------------------
Ran 2 tests in 0.276s

FAILED (errors=2)

Destroying test database for alias 'default'...
dev ➜ /workspace $ 

`

Reply:
Changing the parameters to be optional would solve the issue, but would not allow making tests that actually test that feature. On the other hand I do not think there is test currently that tests that code yet.

Reply:
And in my clone of your PR, if I add at True argument to the two lines above, the tests pass without error.